taxcalc/tests/test_dropq.py

```diff
@@ -33,10 +33,24 @@
   "behavior": {
     "_BE_sub": {"2016": [0.25]}
   },
+  "consumption": {
```

Comment:
Yes, should be in ASSUMP_CONTENTS.  Thanks for finding this problem, which I've fixed in commit 4e5af19.